js/ui/map.js

```diff
@@ -985,6 +985,21 @@ util.extend(Map.prototype, /** @lends Map.prototype */{
     },
 
     /**
+     * Returns a Boolean indicating whether the map is currently in a "moving" state.
+     * Returns `false` if zooming, rotating, pitching, and dragging are all inactive.
+     *
+     * @returns {boolean} A Boolean indicating whether the map is moving.
+     */
+    isMoving: function() {
+        if (this.zooming ||
+            this.rotating ||
+            this.pitching ||
+            this.dragPan._active ||
+            this.dragRotate._active) return true;
```

Comment:
> The only way I was able to capture "pan" and "rotate" actions was through those `dragPan` and `dragRotate` parameters, though. Are there other places to look for them?

We may need to create the place to look for this state by

 - creating `map.panning` property
 - ensuring that `map.rotating` and `map.rotating` works with the `dragRotate` handler

> I'm not sure - any chance you could provide a case where this happens?

Imagine I click and drag the map by 5 pixels, initiating a "drag pan" gesture, put a rock on my mouse button, and go for lunch. `Map#isMoving` will be `true` the whole time I'm gone but the map won't be moving. Is that the behaviour we want? I'm not sure. It deserves some discussion.